artifactory/commands/generic/upload.go

```diff
@@ -178,30 +179,23 @@ func getMinChecksumDeploySize() (int64, error) {
 	return minSize * 1000, nil
 }
 
-func addBuildProps(props *string, buildName, buildNumber string) error {
-	if buildName == "" || buildNumber == "" {
-		return nil
-	}
+func addBuildProps(buildName, buildNumber string) (props string, err error) {
```

Comment:
The name of this function is no longer accurate, because the function no longer "adds" build props.
Actually, thus function has become completely redundant - you can just remove it, and call the API it includes directly.